Sources/Tests/CoreTests/test_plCmdParser.cpp

```diff
@@ -547,3 +547,31 @@ TEST(plCmdParser, case_sensitive_match)
 
     EXPECT_EQ(specified, true);
 }
+
+TEST(plCmdParser, separator_in_value_nix)
+{
+    const plCmdArgDef cmds[] = {
+        { kCmdTypeString, "color", 0}
+    };
+
+    plCmdParser parser(cmds, std::size(cmds));
+    parser.Parse("plCmdParser --color=reddish-blue");
+
+    ST::string value = parser.GetString("color");
+
+    EXPECT_EQ(value, "reddish-blue");
+}
+
+TEST(plCmdParser, separator_in_value_windows)
+{
+    const plCmdArgDef cmds[] = {
+        { kCmdTypeString, "color", 0}
+    };
+
+    plCmdParser parser(cmds, std::size(cmds));
+    parser.Parse("plCmdParser /color reddish-blue");
```

Comment:
I'm not exactly sure what the specific syntax supported by Uru is here. Starting with the Windows "/" and no space syntax since that seems to be what most older Windows apps expect. (Newer ones are using the *nix convention which would also work on Windows.)

Reply:
I know for sure it does stuff like `/Renderer=gl` but not sure about `/Renderer gl`

Reply:
I can throw the = in. It should all be ok according to the parser, but probably best to stick to what is being documented or typically used.